src/axom/inlet/examples/lua_library.cpp

```diff
@@ -11,6 +11,16 @@
 #include "axom/inlet.hpp"
 #include "axom/slic/core/SimpleLogger.hpp"
 
+// _inlet_sol_state_start
+#include "axom/sol.hpp"

```

Comment:
Suggestion: Would it make sense to add a comment about this header inclusion being necessary?